src/ngraph/runtime/gpu/CMakeLists.txt

```diff
@@ -22,30 +22,39 @@ set(SRC
     cuda_emitter.cpp
     cudnn_emitter.cpp
     cublas_emitter.cpp
+    host_emitter.cpp
     gpu_backend.cpp
+    gpu_call_frame.cpp
     gpu_cuda_context_manager.cpp
     gpu_cuda_function_builder.cpp
     gpu_cuda_function_pool.cpp
     gpu_cuda_kernel_builder.cpp
     gpu_emitter.cpp
-    gpu_external_function.cpp
+    gpu_compiled_function.cpp
+    gpu_internal_function.cpp
     gpu_invoke.cpp
+    gpu_kernel_args.cpp
     gpu_kernel_emitters.cpp
     gpu_memory_manager.cpp
     gpu_primitive_emitter.cpp
+    gpu_runtime_constructor.cpp
     gpu_runtime_context.cpp
     gpu_tensor_wrapper.cpp
     gpu_tensor.cpp
     gpu_util.cpp
     type_info.cpp
+    pass/gpu_batch_norm_cache.cpp
     pass/gpu_layout.cpp
-    pass/tensor_memory_reservation.cpp
-    gpu_kernel_args.cpp
     pass/gpu_rnn_fusion.cpp
-    pass/gpu_batch_norm_cache.cpp
+    pass/tensor_memory_reservation.cpp
     op/batch_norm.cpp
     op/rnn.cpp
     )
+
+if (NOT NGRAPH_DEX_ONLY)
```

Comment:
I'm not a fan of this but ok for now. We should really be building two backends, or just pick one and run with it. But we can address this later. for now this is ok but think about how we can do this better